src/csexp_rpc/csexp_rpc.ml

```diff
@@ -290,7 +290,8 @@ module Client = struct
       in
       Worker.stop async;
```

Comment:
Uh, I don't know if it's a problem, but calling `Worker.stop` before you're done with the worker looks weird, no?
Anything wrong with doing it as before, but moving the "t.async <- None" line up?

Reply:
The only thing `Worker.stop` does is forbid additional `Worker.add_work` calls. Maybe it should be renamed if that's confusing?

> Anything wrong with doing it as before, but moving the "t.async <- None" line up?

Sure, though it makes no difference really.

Reply:
Hum.. I claim that idiomatic use of Fiber is to put the side-effects inside the fiber when possible, so `add_work` doing the side-effect (checking if worker is stopped or not) outside of the fiber is not a great API, and it's even worse when we're relying on it.

But this is a stylistic thing, so let's get back to it when we run out of straight-up bugs. :-D